src/components/QandA/QandA.jsx

```diff
@@ -1,64 +1,263 @@
-import React, { Component, useState, useEffect } from 'react';
+import React, { useState, useEffect, useContext } from 'react';
+import axios from 'axios';
+import styled from 'styled-components';
 import QuestionList from './QuestionList.jsx';
 import './QAstyles/QandA.css';
+import MagGlass from "./searchicon.png"
 import server from '../../serverRequests.js';
 
-const QandA = ({prod}) => {
-  const [search, setSearch] = useState('');
-  const [count, setCount] = useState(1);
-  const [questions, setQuestions] = useState(['Question_1', 'Question_2', 'Question_3', 'Question_4'])
+//------------------------ styled-components ------------------------//
+const Button = styled.button`
+  background: transparent;
+  border-radius: 3px;
+  border: 2px solid grey;
+  margin-right: 1em;
+  padding: 0.5em 1em;
+  &:hover {
+    background: lightgrey;
+    box-shadow: rgba(0, 0, 0, 0.25) 0px 5px 10px;
+    transform: scale(1.05);
+  }
+`;
+
+const Questions = styled.section`
+  overflow: auto;
+  height: 100%;
+  max-height: 85vh;
+  width: auto;
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+`;
+
+const Sort = styled.div`
+  padding: 1em;
+  margin: 0 auto;
+`;
 
-  //axios
-  const data = () => {
-    return console.log('prod = ', prod)
+const Search = styled.div`
+  display: flex;
+  line-height: 24px;
+  margin-right: 10px;
+  border: 1px solid black;
+  border-radius: 5px;
+  vertical-align: middle;
+  width: 60%;
+  @media (max-width: 425px) {
+    width: 90%;
   }
+`;
+
+const Input = styled.input`
+  display: inline-block;
+  vertical-align: middle;
+  font-size: 12px;
+  text-align: middle;
+  border: 1px solid grey;
+  height: 40px;
+  width: 100%;
+  &:focus {
+    outline: none;
+  }
+  @media (max-width: 425px) {
+    font-size: 10px
+  }
+`;
+
+const Icon = styled.img`
+  position: absolute;
+  right: 40%;
+  width: 25px;
+  height: 25px;
+  margin: 10px;
+`;
+//-------------------------------------------------------------------//
+
+function QandA({ prod }) {
+
+  const [search, setSearch] = useState('');
+  const [count, setCount] = useState({ questions: 1, answers: 1 });
+
+  // ------------- product --------------------
+  const [productID, setProductID] = useState('');
+  const [results, setResults] = useState('');
+  const [showData, setShowData] = useState([
+    // {questionID: '', question: '', answers: []}
+  ]);
+  const [questionCount, setQuestionCount] = useState(1);
```

Comment:
BRDs ask for default of 4, unless you're still using 1 as default for functionailty testing.

Reply:
Yes, to test the "More Answered Questions" button